src/cluster_legacy.c

```diff
@@ -916,6 +974,35 @@ static void updateAnnouncedHumanNodename(clusterNode *node, char *new) {
     clusterDoBeforeSleep(CLUSTER_TODO_SAVE_CONFIG);
 }
 
+static void updateAnnouncedClientIpV4(clusterNode *node, char *new) {
+    if (new && !strcmp(new, node->client_ipv4)) {
```

Comment:
`new` is a reserved keyword in c++ and it confuses clang-format 18 in some cases. 

I noticed that this also comes from `hostname` - can we fix that too? 

```suggestion
    if (new_ipv4 && !strcmp(new, node->client_ipv4)) {
```